ci-operator/step-registry/ipi/install/heterogeneous/ipi-install-heterogeneous-ref.yaml

```diff
@@ -17,5 +20,15 @@ ref:
   - name: ADDITIONAL_WORKER_VM_TYPE
     default: "m6g.xlarge"
     documentation: "VM Instance type for Heterogeneous worker, default set to arm64 instance type"
+  - name: CLOUD_TYPE
+    default: "aws"
+    documentation: |-
+      The cloud platform where the cluster is installed, this is added for claiming cluster from a cluster pool, if
+      the cluster is a fresh install with cluster_profile, will using CLUSTER_TYPE value.
```

Comment:
```suggestion
      The cloud platform where the cluster is installed. This is needed when using cluster_claims. Otherwise, CLUSTER_TYPE is used.
```